client/components/UserSettings/index.jsx

```diff
@@ -1,14 +1,86 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Checkbox from '@components/Checkbox';
+import { handleGetValidAts } from '../../actions/ats';
```

Comment:
Suggestion: make a module that imports and re-exports only the components that your application needs, then import from _that_ module: 

(`client/components/UI/index.js`)
```js
import Button from 'react-bootstrap/Button';
import Checkbox from '@components/Checkbox';
import Col from 'react-bootstrap/Col';
import Form from 'react-bootstrap/Form';
import Row from 'react-bootstrap/Row';

export {
  Button,
  Checkbox,
  Col,
  Form,
  Row,
};
```

Then in your other components: 

```js
import {
  Button,
  Checkbox,
  Col,
  Form,
  Row
} from '@components/UI';
```


(Then you can even group your custom components in that single module, as I've done here with `Checkbox`)